config/zones.json

```diff
@@ -3708,6 +3708,31 @@
     },
     "timezone": null
   },
+  "US-SEC": {
```

Comment:
do we already have a geometry for this?

Reply:
sadly, no. that's my next project!

Reply:
ok, then let's wait to merge this PR for when the geometry is ready

Reply:
sure. i may need a bit of help, though. details in https://github.com/tmrowco/electricitymap-contrib/issues/1713#issuecomment-513506553 . if you know who i should talk to, please point me their way!